packages/layout/src/tree-layout/layout-definition/definition-manager/definitionModel.js

```diff
@@ -0,0 +1,60 @@
+export default class DefinitionModel {
+    constructor (host, cut, ratioWeight, preferred, lanes) {
```

Comment:
No more than four arguments in a constructor/method/function.
Group params logically to restrict it to 4